lib/Backends/OpenCL/CMakeLists.txt

```diff
@@ -1,4 +1,30 @@
+add_custom_command(
+  OUTPUT "${CMAKE_BINARY_DIR}/glow/kernels.inc"
+  COMMAND include-bin 
+          "${CMAKE_CURRENT_SOURCE_DIR}/kernels.cl"
+          "${CMAKE_BINARY_DIR}/glow/kernels.inc"
+  DEPENDS include-bin CPURuntime "${CMAKE_CURRENT_SOURCE_DIR}/kernels.cl")
+
+add_custom_command(
+  OUTPUT "${CMAKE_BINARY_DIR}/glow/kernels_fwd_conv.inc"
+  COMMAND include-bin
+          "${CMAKE_CURRENT_SOURCE_DIR}/kernels_fwd_conv.cl"
+          "${CMAKE_BINARY_DIR}/glow/kernels_fwd_conv.inc"
+  DEPENDS include-bin CPURuntime
+          "${CMAKE_CURRENT_SOURCE_DIR}/kernels_fwd_conv.cl")
+
+add_custom_command(
+  OUTPUT "${CMAKE_BINARY_DIR}/glow/kernels_fwd_quantized_conv.inc"
```

Comment:
Would it make sense to let the xxd utility take multiple arguments and just concat them into a single array?

Reply:
Yeah, I think it could be useful.

Reply:
But for our OpenCL use of this utility we don't need it, because we compile those string arrays using the OpenCL driver at different points during the execution.

Reply:
Okay.